app/controllers/book_controller.rb

```diff
@@ -0,0 +1,13 @@
+class BookController < ApplicationController
```

Comment:
Ok, that's good, but we will separating what constitue the API than other controller. 
Also I would like you use [versionist](https://github.com/bploetz/versionist) gem.